.circleci/config.yml

```diff
@@ -77,7 +77,7 @@ jobs:
       - deploy:
           name: Maybe push release image
           command: |
-            if echo "${CIRCLE_TAG}" | grep -Eq "^[0-9]+(\.[0-9]+)*(-[a-z]+)?$"; then
+            if echo "${CIRCLE_TAG}" | grep -Eq "[0-9]+(\.[0-9]+)*(-[a-z0-9]+)?"; then
```

Comment:
It should be the previous (full match) regex with only `0-9` added, as this will also match e.g. `foo-1.13.0-foo`.

```sh
$ if echo "1.0.0-rc1" | grep -Eq "[0-9]+(\.[0-9]+)*(-[a-z]+)?"; then echo "TRUE"; fi
TRUE

$ if echo "1.0.0-rc1" | grep -Eq "[0-9]+(\.[0-9]+)*(-[a-z0-9]+)?"; then echo "TRUE"; fi
TRUE

$ if echo "1.0.0-rc1" | grep -Eq "^[0-9]+(\.[0-9]+)*(-[a-z]+)?$"; then echo "TRUE"; fi

$ if echo "foo-1.0.0-rc1" | grep -Eq "[0-9]+(\.[0-9]+)*(-[a-z0-9]+)?"; then echo "TRUE"; fi
TRUE
```